src/lib/source-file-converter.ts

```diff
@@ -834,14 +885,41 @@ export class SourceFileConverter {
     );
   }
 
+  /**
+   * Follow all aliases, such as imports, for the given node. Return the original
+   * symbol. If node is not an identifier, this method will return `undefined`.
+   *
+   * @param node The node to check. It can be any node type but this will return
+   *     undefined for non-Itentifiers.
+   */
+  getOriginalSymbol(node: ts.Node): ts.Symbol | undefined {
```

Comment:
Do we call this in any place where `undefined` is an acceptable return value? Should we throw/report instead?